cmake/configure.cmake

```diff
@@ -82,11 +82,12 @@ if(WITH_GOLANG)
   file(MAKE_DIRECTORY "${PADDLE_IN_GOPATH}")
   set(PADDLE_GO_PATH "${CMAKE_SOURCE_DIR}/go")
 
-  add_custom_target(go_path)
-  add_custom_command(TARGET go_path
+  message(STATUS "link ${CMAKE_SOURCE_DIR} to ${PADDLE_IN_GOPATH}...")
+
+  execute_process(
```

Comment:
Interesting, don't know why travis didn't run successfully. Well, if compiling with cmake, the vendor will be prepared at `cmake` time, and following `go build` will pass. Seems https://github.com/PaddlePaddle/Paddle/pull/2884/ is better~ Thanks~